### PR TITLE
Clean up magic numbers

### PR DESCRIPTION
**Pull request by NotWoods**
Small refactorings for functions that used magic numbers to try and use the suppression less.

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: components/service/telemetry/src/main/java/org/mozilla/telemetry/net/TelemetryClient.kt [hunk 1]
```diff
@@ -45,13 +45,13 @@ class TelemetryClient(
 
         logger.debug("Ping upload: $status")
 
-        when (status) {
+        return when (status) {
```
[the remark this answers is not in the document]
Comment (NotWoods) [reply components/service/telemetry/src/main/java/org/mozilla/telemetry/net/TelemetryClient.kt]: Turns out you can! I moved `@Suppress` for some methods.

**Comment by NotWoods** [comment]
I've turned the lint rule option back on and changed the focus of this PR to just use fewer magic numbers.

File: components/feature/qr/src/main/java/mozilla/components/feature/qr/QrFragment.kt [hunk 2]
```diff
@@ -269,14 +271,12 @@ class QrFragment : Fragment() {
...
-            var swappedDimensions = false
-            when (displayRotation) {
-                Surface.ROTATION_0, Surface.ROTATION_180 -> if (sensorOrientation == 90 || sensorOrientation == 270) {
-                    swappedDimensions = true
-                }
-                Surface.ROTATION_90, Surface.ROTATION_270 -> if (sensorOrientation == 0 || sensorOrientation == 180) {
-                    swappedDimensions = true
-                }
+
+            @Suppress("MagicNumber")
```
Comment (jhugman) [inline components/feature/qr/src/main/java/mozilla/components/feature/qr/QrFragment.kt]: 😍

File: components/service/telemetry/src/main/java/org/mozilla/telemetry/net/TelemetryClient.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhugman) [reply components/service/telemetry/src/main/java/org/mozilla/telemetry/net/TelemetryClient.kt]: Perfect, thanks!

File: samples/toolbar/src/main/java/org/mozilla/samples/toolbar/ToolbarActivity.kt [hunk 3]
```diff
@@ -469,7 +469,7 @@ class ToolbarActivity : AppCompatActivity() {
                         view.progress = progress
                     }
 
-                    delay(progress * 5L)
+                    delay(progress * RELOAD_STEP_SIZE.toLong())
```
Comment (jhugman) [inline samples/toolbar/src/main/java/org/mozilla/samples/toolbar/ToolbarActivity.kt]: Ok.

**Pull request merged**